src/keys/jwk2pem.ts

```diff
@@ -3,14 +3,14 @@ import 'node-forge/lib/rsa.js'
 import forge from 'node-forge/lib/forge.js'
 import { base64urlToBigInteger } from '../util.js'
 
-function convert (key: any, types: string[]) {
+function convert (key: any, types: string[]): any {
```

Comment:
Is it possible to lose the `any`?  I think the type should be `forge.jsbn.BigInteger[]`?